tracker/dmlc_tracker/local.py

```diff
@@ -43,7 +46,7 @@ def exec_cmd(cmd, role, taskid, pass_env):
             if num_retry >= 0:
                 # failure trail increase by 1 and restart failed worker
                 for arg in cmd:
-                    if arg.startswith('rabit_num_trial'):
+                    if arg.startswith('DMLC_NUM_ATTEMPT'):
```

Comment:
it's very weird to have this in local.py, this is essentially a param only for test, and particularly only for any test requiring to indicate how many times the process has been started

Reply:
the expected solution is to make dmlc-submit as spark-submit, it takes several parameters for its own, like --num-executors, and anything following the path to a jar should be taken as the application-specific params, and they are delegate to the application itself to process